code/_global_vars/roundstats.dm

```diff
@@ -0,0 +1,78 @@
+
+//This is for the round end stats system.
+
+//roundstat is used for easy finding of the variables, if you ever want to delete all of this,
+//just search roundstat and you'll find everywhere this thing reaches into.
+//It used to be bazinga but it only fly with microwaves.
+
+GLOBAL_VAR_INIT(lights_switched_on_roundstat, 0)
+GLOBAL_VAR_INIT(cans_opened_roundstat, 0)
+GLOBAL_VAR_INIT(turbo_lift_floors_moved_roundstat, 0)
+GLOBAL_VAR_INIT(lost_limbs_shift_roundstat, 0)
+GLOBAL_VAR_INIT(seed_planted_shift_roundstat, 0)
+GLOBAL_VAR_INIT(step_taken_shift_roundstat, 0)
+GLOBAL_VAR_INIT(destroyed_research_items_roundstat, 0)
+GLOBAL_VAR_INIT(items_sold_shift_roundstat, 0)
+GLOBAL_VAR_INIT(disposals_flush_shift_roundstat, 0)
+GLOBAL_VAR_INIT(rocks_drilled_roundstat, 0)
+
+//Virgo specific
+//GLOBAL_VAR_INIT(mouse_spawned_shift_roundstat, 0)
+//GLOBAL_VAR_INIT(number_people_walked_over_roundstat, 0)
+//GLOBAL_VAR_INIT(trash_piles_searched_roundstat, 0)
+//GLOBAL_VAR_INIT(rare_trash_found_roundstat, 0)
+
+
+/hook/roundend/proc/RoundEnd()
+
+	var/lights_switched_on = GLOB.lights_switched_on_roundstat
```

Comment:
Why are you saying "This variable = That variable", why not just output GLOB.lights_switched_on_roundstat?

Reply:
Last time i tried, i had issues where the variables wouldn't actually update. But it may not be necessary.